src/Pages/ContentListTable/components/SnapshotListModal/SnapshotListModal.tsx

```diff
@@ -245,11 +251,25 @@ export default function SnapshotListModal() {
                           }
                         />
                       </Td>
-                      <Td>{content_counts?.['rpm.package'] || 0}</Td>
+                      <Td>
+                        <Button
+                          onClick={() => {
+                            navigate(`${rootPath}/${uuid}/snapshots/${snap_uuid}`);
+                          }}
+                          style={{
+                            color: 'lightblue',
+                            border: 'none',
+                            background: 'none',
+                            padding: 0,
+                          }}
```

Comment:
You don'e need to create a custom style here (or shouldn't at least). 
You can look at the 'Inline link' variant found [here](https://www.patternfly.org/components/button#variant-examples).